scripts/vs/buildimage-vs-image/test.sh

```diff
@@ -43,5 +44,9 @@ if [ $? != 0 ]; then
         sudo chown -R johnar.johnar kvmdump
         virsh -c qemu:///system undefine $dut
     fi
+
+    docker commit $ptf_name docker-ptf:${JOB_NAME##*/}.${BUILD_NUMBER}
+    docker save docker-ptf:${JOB_NAME##*/}.${BUILD_NUMBER} | gzip -c > docker-ptf-dump.gz
```

Comment:
do we know if this will be archived?

Reply:
suggest to create a folder called ptfdump and put into the folder.

Reply:
you need to change the jenkinsfile to add this pfdump folder in the archive list.

Reply:
also can you check if commit or save leave footprint in the docker file system, if yes, we need to clean up them, otherwise the docker filesystem will grow.

Reply:
ack, good catch!